spec/schema.json

```diff
@@ -0,0 +1,32 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Companies House - Limited Partnerships API",
+    "description": "An API to allow the registration, transition and update of limited partnerships",
+    "version": "1.0.0"
+  },
+  "host": "api.companieshouse.gov.uk",
+  "tags": [
+    {
+      "name": "Limited Partnerships"
+    }
+  ],
+  "paths": {
+    "/limited-partnership/healthcheck": {
+      "post": {
```

Comment:
Needs changing to "get". Also the summary needs updating.

Reply:
Updated to say "get"

Reply:
The summary for the healthcheck still needs updating. It's not creating a registration submission. Suggest something like: `Health check URL returns 200 if service is running`

Reply:
Updated the summary.